weather_mv/loader_pipeline/util.py

```diff
@@ -117,6 +117,8 @@ def to_json_serializable_type(value: t.Any) -> t.Any:
 
         # We assume here that naive timestamps are in UTC timezone.
         return value.replace(tzinfo=datetime.timezone.utc).isoformat()
+    elif isinstance(value, datetime.timedelta):
```

Comment:
Please add test case for the same.

Reply:
added in [18d632c](https://github.com/google/weather-tools/pull/349/commits/18d632c2af4aa587886c12312af1cdf7dc0e57db)